### DISCUSSION TITLE
how to get a tileset image from a blender file ?

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#963] [General]
Hello,
looking at the opengameart tilesets I saw these magnificent bridge tilesets, only once in Blender I can't get an image rendering of the tiles as well as the lighting and textures. I would like to know how to get an image with the tiles broken down from the blender file:

https://opengameart.org/sites/default/files/FLARE_Bridge_0.zip

**Comment by dorkster** [comment]
The missing lighting and textures are due to those tiles being made in Blender 2.59. Blender changed a lot of things with version 3 onward that made it incompatible with older files. The easiest path would be to just use an older version of Blender to open the file. If not, you would need to manually fix the lighting and textures for it to work with one of Blender's current renderers (either EEVEE or Cycles).

Once you do get the tiles rendered, you can use ImageMagick's `montage` program from the command line to put all the tiles in a single sheet:

```
montage -background transparent -geometry 64x192 -tile 8x8 Tiles/*.png tileset_bridge.png
```
